### PR TITLE
[Sprint 4 ]Feature Join Domain Shop Management

### PR DESCRIPTION
**Pull request by tuanlvt-1117**

### CODE REVIEW

File: app/src/main/res/drawable-v21/ic_location.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (daolq3012) [inline app/src/main/res/drawable-v21/ic_location.xml]: WARNING: [UnusedResources] The resource `R.drawable.ic_location` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/layout/item_join_domain.xml [hunk 2]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (daolq3012) [inline app/src/main/res/layout/item_join_domain.xml:2]: WARNING: [UnusedResources] The resource `R.layout.item_join_domain` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -95,4 +95,10 @@
     <string name="reject">Reject</string>
     <string name="select_all">Select all</string>
     <string name="see_more">See More..</string>
+    <string name="user">User :</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:98]: WARNING: [UnusedResources] The resource `R.string.user` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -95,4 +95,10 @@
...
+    <string name="products">Product :</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:99]: WARNING: [UnusedResources] The resource `R.string.products` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -95,4 +95,10 @@
...
+    <string name="shops">Shop :</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:100]: WARNING: [UnusedResources] The resource `R.string.shops` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -95,4 +95,10 @@
...
+    <string name="button_join_domain">button Join Domain</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:101]: WARNING: [UnusedResources] The resource `R.string.button_join_domain` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -95,4 +95,10 @@
...
+    <string name="image_shop_management">Image Shop Management</string>
+    <string name="domain_location">Domain Location</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:103]: WARNING: [UnusedResources] The resource `R.string.domain_location` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/drawable-v21/ic_location.xml (hunk 1 above)
Comment (tuanlvt-1117) [reply app/src/main/res/drawable-v21/ic_location.xml]: Will use in logic

File: app/src/main/res/layout/item_join_domain.xml (hunk 2 above)
Comment (tuanlvt-1117) [reply app/src/main/res/layout/item_join_domain.xml:2]: Will use in logic

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:98]: Will use in logic

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:99]: Will use in logic

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:101]: Will use in logic

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:103]: Will use in logic

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -95,4 +95,10 @@
     <string name="reject">Reject</string>
     <string name="select_all">Select all</string>
     <string name="see_more">See More..</string>
+    <string name="user">User :</string>
+    <string name="products">Product :</string>
+    <string name="shops">Shop :</string>
+    <string name="button_join_domain">button Join Domain</string>
+    <string name="image_shop_management">Image Shop Management</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:102]: WARNING: [UnusedResources] The resource `R.string.image_shop_management` appears to be unused
 Unused resources make applications larger and slow down builds.

Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:102]: Will use in logic

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (tuanlvt-1117) [reply app/src/main/res/values/strings.xml:100]: Will use in logic

File: app/src/main/res/drawable/ic_location.xml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (daolq3012) [inline app/src/main/res/drawable/ic_location.xml:2]: WARNING: [UnusedResources] The resource `R.drawable.ic_location` appears to be unused
 Unused resources make applications larger and slow down builds.

Comment (tuanlvt-1117) [reply app/src/main/res/drawable/ic_location.xml:2]: Will use in logic

**Pull request merged**